deep_qa/layers/memory_updaters.py

```diff
@@ -53,41 +53,34 @@ class DenseConcatNoQuestionMemoryUpdater(Dense):
     Because the input to the memory updater is already concatenated, we just remove the question
     representation and then send this through the Dense layer.
     """
-    def __init__(self, encoding_dim, name="dense_concat_memory_updater", **kwargs):
-        self.encoding_dim = encoding_dim
-        super(DenseConcatNoQuestionMemoryUpdater, self).__init__(encoding_dim, name=name, **kwargs)
+    def __init__(self, output_dim, name="dense_concat_memory_updater", **kwargs):
+        super(DenseConcatNoQuestionMemoryUpdater, self).__init__(output_dim, name=name, **kwargs)
 
     def call(self, x, mask=None):
         # For this memory update, we don't need the question encoding. This is the first
         # concatentated vector, so we remove it.
-        x = x[:, self.encoding_dim:]
+        x = x[:, self.output_dim:]
         return super(DenseConcatNoQuestionMemoryUpdater, self).call(x)
 
     def get_output_shape_for(self, input_shape):
         return (input_shape[0], int(input_shape[1] / 3))
 
-    def get_config(self):
```

Comment:
It's inheriting from `Dense`, which has a `get_config` method, and doesn't add any new parameters.